requests/request_body.go

```diff
@@ -29,9 +30,30 @@ type RequestBodyValidator interface {
 	ValidateRequestBodyWithPathItem(request *http.Request, pathItem *v3.PathItem, pathValue string) (bool, []*errors.ValidationError)
 }
 
+type configOptions struct {
+	regexEngine jsonschema.RegexpEngine
+}
+
+// Option supports the 'Options Pattern' to define the behavior of a RequestBodyValidator
+type Option func(options *configOptions)
+
+// WithRegexEngine allows for a custom regular expression engine to be used during validation.
+func WithRegexEngine(engine jsonschema.RegexpEngine) Option {
+	return func(rbv *configOptions) {
+		rbv.regexEngine = engine
+	}
+}
+
 // NewRequestBodyValidator will create a new RequestBodyValidator from an OpenAPI 3+ document
-func NewRequestBodyValidator(document *v3.Document) RequestBodyValidator {
-	return &requestBodyValidator{document: document, schemaCache: &sync.Map{}}
+func NewRequestBodyValidator(document *v3.Document, opt ...Option) RequestBodyValidator {
+
+	cfg := configOptions{} // Default Options
+	for _, o := range opt {
```

Comment:
```suggestion
func NewRequestBodyValidator(document *v3.Document, opts ...Option) RequestBodyValidator {

	cfg := configOptions{} // Default Options
	for _, opt := range opts {
```

As you did everywhere else

Reply:
your `opts` variadic is named `opts`, except here

the the loop here is on a variable named `o`, while everywhere else it's `opt`

So, I'm simply asking to apply the suggestion above and write

```go
func NewRequestBodyValidator(document *v3.Document, opts ...Option) RequestBodyValidator {
	cfg := configOptions{} // Default Options
	for _, opt := range opts {
```